calm/control-example/control-evidence.json

```diff
@@ -0,0 +1,14 @@
+{
+  "$schema": "control-specification.json",
+  "$id": "control-evidence.json",
+  "title": "Evidence of pre-production review",
+  "control-id": "ci-arch-001",
+  "data-fields": {
+    "review-date": "2024-08-02",
+    "reviewed-by": "jpgough-ms",
+    "docs-link": "https://issue-tracker.com/12345"
+  },
+  "name": "Architecture review pre-production",
+  "scope-text": "All workloads going to production",
+  "scope-rego": "input.metadata.target-deployment.environment == Production"
```

Comment:
This should be in the control instance rather than the evidence of that control for a given system.

Reply:
I think I'm still confused about the relationship between control, instance. Maybe `control-evidence` can just be renamed to `control-instance`?